cmd/wallet-js-sdk/scripts/setup_test.sh

```diff
@@ -8,8 +8,6 @@ set -e
 
 
 if [ "$1" == "setup" ]; then
```

Comment:
we are not generating keys anymore? or moved?

Reply:
done